app/src/main/java/com/example/androidthings/assistant/AssistantActivity.java

```diff
@@ -192,9 +206,9 @@ public void onConversationFinished() {
                         Log.i(TAG, "assistant conversation finished");
                         if (mLed != null) {
                             try {
-                                mLed.setValue(false);
+                                mLed.setValue(true);
                             } catch (IOException e) {
-                                Log.e(TAG, "error turning off LED:", e);
+                                Log.w(TAG, "error turning off LED:", e);
```

Comment:
I think keeping the level as error is preferable.

Reply:
sure

Reply:
Did you make the change?

Reply:
mmm, strange, it shows Log.i on my local machine and the git diff shows nothing.

Reply:
Sorry I missed that one. Fixed now